test/integration/integration_test_fixtures.py

```diff
@@ -141,34 +150,26 @@ def _tryStartTestServers(self):
     return True
 
   def getGlobalResults(self, parsed_json):
-    """
-    Utility to find the global/aggregated result in the json output
-    """
+    """Find the global/aggregated result in the json output."""
     global_result = [x for x in parsed_json["results"] if x["name"] == "global"]
     assert (len(global_result) == 1)
     return global_result[0]
 
   def getNighthawkCounterMapFromJson(self, parsed_json):
-    """
-    Utility method to get the counters from the json indexed by name.
-    """
+    """Get the counters from the json indexed by name."""
     return {
         counter["name"]: int(counter["value"])
         for counter in self.getGlobalResults(parsed_json)["counters"]
     }
 
   def getNighthawkGlobalHistogramsbyIdFromJson(self, parsed_json):
-    """
-    Utility method to get the global histograms from the json indexed by id.
-    """
+    """Get the global histograms from the json indexed by id."""
     return {
         statistic["id"]: statistic for statistic in self.getGlobalResults(parsed_json)["statistics"]
     }
 
   def getTestServerRootUri(self, https=False):
-    """
-    Utility for getting the http://host:port/ that can be used to query the server we started in setUp()
-    """
+    """Get the http://host:port/ that can be used to query the server we started in setUp()."""
```

Comment:
(nit) The one-line docstring should fit on a single line (80 chars).

If it doesn't (i.e. we need to say more) we should still choose an abstract that fits into 80 charts and write the rest into the body of the docstring.

Applies in a few places.